packages/core/content-releases/admin/src/services/release.ts

```diff
@@ -1,4 +1,5 @@
 import { createApi } from '@reduxjs/toolkit/query/react';
+import get from 'lodash/get';
```

Comment:
Do you think you could access what you need without lodash?

Reply:
Done 👍